typescript/infra/config/environments/mainnet3/warp/configGetters/getPulsechainUSDCWarpConfig.ts

```diff
@@ -49,7 +42,9 @@ const getRebalanceableCollateralTokenConfigForChain = (
     `USDC token address not found for chain ${currentChain}`,
   );
 
-  const currentRebalancingConfig = rebalancingConfigByChain[currentChain];
+  const currentRebalancingConfig = getUSDCRebalancingBridgesConfigFor(
+    Object.keys(ownersByChain),
+  )[currentChain];
```

Comment:
The only disadvantage to this that i see is that now for every collateral chain we are going to regenerate this from scratch, probably i would find a way to avoid this. Maybe you could preload the config as it was done previously and pass the resulting mapping as a param to the `getRebalanceableCollateralTokenConfigForChain` function